styles.css

```diff
@@ -0,0 +1,42 @@
+body {
+  background-color: #E96D65;
+  margin: 50px;
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  font-size: 2.5em;
+}
+.input {
+  position: relative;
+}
+input {
+  background-color: #E96D65;
+  border: none;
+  border-bottom: 1px dotted #b14943;
+  color: #b14943;
+}
+
+input[type=number]:focus {
```

Comment:
Since all your inputs are `[type=number]`, there is no need to have these styles in a separate rule, you can just do:

```css
input:focus {
```